src/ui/qgisapp.ui

```diff
@@ -2654,7 +2662,11 @@ Shift+click to snap rotation to 45 degree steps.</string>
     <string>Fill Ring</string>
    </property>
    <property name="toolTip">
-    <string>Fill Ring</string>
+    <string>Fill Ring
+Creates a hole in a polygon and fills it with a new feature:
+Click within an existing polygon to draw new inner ring.
+Right click to finish and fill ring.
```

Comment:
```suggestion
Left clicks + right click within an existing polygon to draw a new filled inner ring.
```
I'm not convinced by my description but the idea is to merge these two rows and clarify they are in the same single process. And it would pair with the other option which is also a whole process by itself.
_note: At first reading I naively thought I could just click within the polygon to get the ring (😞)._

Reply:
Maybe-
Click+click to draw a new inner ring in a polygon and right click to finish and fill ring.